master/static/migrations/20230920130838_add-update-modified-at-in-user-table-by-insert.tx.up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TRIGGER autoupdate_users_modified_at
```

Comment:
sorry, i thought this had a different name. there is already a trigger with this name.